be/src/formats/orc/apache-orc/c++/include/orc/Common.hh

```diff
@@ -18,11 +18,14 @@
 
 #pragma once
 
+#include <glog/logging.h>
```

Comment:
remove it?

Reply:
I put a DCHECK at line 232 to check pre-condition.